modules/porous_flow/examples/flow_through_fractured_media/tests

```diff
@@ -29,6 +30,7 @@
     abs_zero = 1E-7
     max_parallel = 1 # without the influence of a time derivative, MOOSE believes the porepressure in the ultra-low permeability matrix can be almost anything: the residual are swamped by the residuals in the fracture
     threading = '!pthreads'
+    mumps = True
```

Comment:
If we add this, should we set the input file to use mumps as well?  Do similar things for all other changes.

`Preconditioning/active=mumps_is_best_for_parallel_jobs`